hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/RouterYarnClientUtils.java

```diff
@@ -194,4 +191,24 @@ private static boolean mergeUamToReport(String appName,
     return !(appName.startsWith(UnmanagedApplicationManager.APP_NAME) ||
         appName.startsWith(PARTIAL_REPORT));
   }
+
+  /**
+   * Merges a list of GetClusterNodesResponse.
+   *
+   * @param responses a list of GetClusterNodesResponse to merge
+   * @return the merged GetClusterNodesResponse
+   */
+  public static GetClusterNodesResponse mergeClusterNodesResponse(
+          Collection<GetClusterNodesResponse> responses) {
+    GetClusterNodesResponse clusterNodesResponse = Records.newRecord(
+            GetClusterNodesResponse.class);
```

Comment:
Check the checkstyle report from Yetus.

Reply:
Thanks, I will check yetus checkstyle report